### PR TITLE
[RFC] vim-patch:7.4.668

### PR DESCRIPTION
**Pull request by fmoralesc**
Problem:    Can't use a glob pattern as a regexp pattern.
Solution:   Add glob2regpat(). (Christian Brabandt)

https://code.google.com/p/vim/source/detail?r=v7-4-668

### CODE REVIEW

File: src/nvim/eval.c [hunk 1]
```diff
@@ -6512,6 +6512,7 @@ static struct fst {
   {"getwinposy",      0, 0, f_getwinposy},
   {"getwinvar",       2, 3, f_getwinvar},
   {"glob",            1, 3, f_glob},
+  {"glob2repath",     1, 1, f_glob2regpat},
```
Comment (justinmk) [inline src/nvim/eval.c]: Shouldn't this be `glob2regpat`?

Comment (fmoralesc) [inline src/nvim/eval.c]: Sure should.

**Comment by justinmk** [comment]
Merged thanks :)

**Comment by fmoralesc** [comment]
Thanks, @justinmk. Sorry I couldn't rebase myself, I'm at work :p

**Pull request closed without merging**